### PR TITLE
Remove texture_multisampled_2d_array builtins

### PR DESCRIPTION
**Pull request by jrprice**
This type does not exist in WGSL.

### CODE REVIEW

**Review by alan-baker — APPROVED** [review]

**Review by kvark — APPROVED** [review]

**Comment by kvark** [comment]
@alan-baker did you want to talk about this more before proceeding? Sorry if I merged this too quickly!

**Comment by alan-baker** [comment]
No, I consider the needs approval stage, needs editor approval is that not the right state?

**Comment by kvark** [comment]
I always treated "Needs Approval" as related to the WG (or the WGSL subgroup). Editors can approve things by just reviewing them, they shouldn't need this label internally.

**Pull request merged**
